x/gamm/pool-models/balancer/pool_suite_test.go

```diff
@@ -857,7 +857,10 @@ func (suite *KeeperTestSuite) TestJoinPool() {
 		tc := tc
 
 		suite.T().Run(tc.name, func(t *testing.T) {
-			pool := createTestPool(t, tc.swapFee, sdk.ZeroDec(), tc.poolAssets...)
+			poolI := createTestPool(t, tc.swapFee, sdk.ZeroDec(), tc.poolAssets...)
```

Comment:
let's instead make the `createTestPool` return balancer right away instead of type casting everywhere